transformer/transformer.go

```diff
@@ -194,16 +194,16 @@ func (t *Transformer) transform(source *ESResponse, highlight bool) SearchRespon
 		Items:        []ContentItem{},
 		ContentTypes: []ContentType{},
 	}
-	var took int = 0
+	var took int
 	for _, response := range source.Responses {
-		for _, doc := range response.Hits.Hits {
-			sr.Items = append(sr.Items, t.buildContentItem(doc, highlight))
+		for i := 0; i < len(response.Hits.Hits); i++ {
+			sr.Items = append(sr.Items, t.buildContentItem(response.Hits.Hits[i], highlight))
 		}
-		for _, bucket := range response.Aggregations.DocCounts.Buckets {
-			sr.ContentTypes = append(sr.ContentTypes, buildContentTypes(bucket))
+		for j := 0; j < len(response.Aggregations.DocCounts.Buckets); j++ {
+			sr.ContentTypes = append(sr.ContentTypes, buildContentTypes(response.Aggregations.DocCounts.Buckets[j]))
 		}
-		for _, suggest := range response.Suggest.SearchSuggest {
-			for _, option := range suggest.Options {
+		for z := 0; z < len(response.Suggest.SearchSuggest); z++ {
```

Comment:
Maybe in keeping with mathematical notation, do `i`, `j` .... `k` instead of `z` ?

Reply:
Fixed in update 1.